### ISSUE TITLE
Expose preset temps for presets in HVAC mode

### ISSUE THREAD

**Issue by MaxWaldorf** [jmcollin78/versatile_thermostat#103]
**Is your feature request related to a problem? Please describe.**
It is rather a missing feature as the values are not exposed by the plugin in AC mode...

**Describe the solution you'd like**
I'm using temp sensors (Danfoss) that can help set a target temperature and I'm coupling that with the schedule plugin...

**Describe alternatives you've considered**
I'd like to set the comfort target as the sensor set temp but I can't do that when heat pump is in AC mode as the preset temp is not exposed by your plugin...

**Additional context**
Could look like:
```
service: versatile_thermostat.set_preset_temperature
data:
    preset: boost
    temperature_hvac: 17.8
    temperature_away_hvac: 15
target:
    entity_id: climate.my_thermostat
```

**Comment by jmcollin78** [comment]
Hello,

I think you can change the ac preset value with the following service call:
```
service: versatile_thermostat.set_preset_temperature
        data:
            preset: boost_ac            ####  notice the _ac suffix here 
            temperature: 17.8
            temperature_away: 15
        target:
            entity_id: climate.thermostat_2
```

I will try this and I let you know.

**Comment by jmcollin78** [comment]
Ok this was not accepted because list of preset was limited to eco, boost, comfort.
This is now fixed in 3.4.
